include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.hpp

```diff
@@ -203,7 +203,10 @@ class Orbit : public Trajectory
     String generateFrameName(const Orbit::FrameType& aFrameType) const;
 
     static Instant GetCrossingInstant(
```

Comment:
nit: should the epoch be optional here?

Reply:
It's required to offset the instants to convert to double. So it's required, I don't want to have a fallback to J2000 or something due to the precision issues.

Reply:
Okay. I missed that it was a private function. Disregard